kubernetes/cmsweb/services/newdqmgui.yaml

```diff
@@ -107,12 +129,18 @@ spec:
         - name: eos-krb
           mountPath: /tmp/krb
         - mountPath: /cvmfs/cms.cern.ch
-          name: cms-pvc
+          name: cms-pvc                    
 #PROD#  - name: logs
 #PROD#    mountPath: /home/cmsusr/dqmgui/dqmgui
         securityContext:
           privileged: true
+          capabilities:
+            add:
+              - SYS_ADMIN
```

Comment:
Running as `SYS_ADMIN` is not desired due to security reason. Can you run this with this?